### PR TITLE
Add Ransack and search form. Searches job titles and company names

### PR DESCRIPTION
**Pull request by bobbyburden**
This addresses #18 but is not ready to merge.

I'm looking for input on how to test this without just testing the external Ransack dependency.

This also needs a bit of design love.
screen shot 2015-06-24 at 8 21 18 pm

### CODE REVIEW

**Comment by bobbyburden** [comment]
I added some style to this. I played with having a "Search" label but it looked cluttered. I ended up with what I have in this screenshot:

screen shot 2015-07-16 at 10 47 13 pm

Let me know if this needs more work before getting merged in.

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -24,6 +25,7 @@ def new
   private
 
   def job_params
+    params.permit(:search)
```
Comment (colinrymer) [inline app/controllers/jobs_controller.rb]: Where is this being used?

Comment (bobbyburden) [inline app/controllers/jobs_controller.rb]: It's not being used any longer. I'll remove.

**Comment by keathley** [comment]
This has been merged.

**Pull request closed without merging**